vertexai/VertexAISnippets/VertexAISnippets.swift

```diff
@@ -443,7 +443,7 @@ class Snippets {
     // [START function_modes]
     let model = VertexAI.vertexAI().generativeModel(
       // Setting a function calling mode is only available in Gemini 1.5 Pro
-      modelName: "{{generic_model_name_initialization}}",
+      modelName: "gemini-1.5-pro",
```

Comment:
Note: I think this is the only one of the snippets that needs `gemini-1.5-pro` and the rest can be `gemini-1.5-flash`.

Reply:
Done.